docs/source/faq.md

```diff
@@ -351,6 +351,13 @@ In some cases, the internal memory type cache can misdetect GPU memory as host
 memory, also leading to invalid memory access. This cache can be disabled by
 setting `UCX_MEMTYPE_CACHE=n`.
 
+#### Seeing the error "provided PTX was compiled with an unsupported toolchain"?
```

Comment:
since it's FAQ, i'd phrase it as
`How to fix the error "provided PTX was compiled with an unsupported toolchain"?`
or
`Why am I getting the error "provided PTX was compiled with an unsupported toolchain"?`